### ISSUE TITLE
How to SetState or make change sensor data

### ISSUE THREAD

**Issue by ebattulga** [helto4real/netdaemon#16]
I want to write sensor. But i cannot change sensor state value. Please see my code
```cs
public class TestSensor : NetDaemonApp
{
    public async override Task InitializeAsync()
    {
        Log("Init!");
        var sensor=Entity("sensor.testsensor").SetState("10").ExecuteAsync();
        Log("done");
        //await SetState("sensor.testsensor",10);
    }
}
```

It works without error. All logs printed. But cannot change sensor value.

**Comment by helto4real** [comment]
Thank you for reporting @ebattulga. Do you get  a state object back from the 
`await SetState("sensor.testsensor", 10);` or null? Also try `await SetState("sensor.testsensor", "10");` to see if it is different?

**Comment by ebattulga** [comment]
`await SetState("sensor.testsensor", "10");` return null

**Comment by ebattulga** [comment]
I'm creating `sensor.testsensor` manually in `Developer tools` > `State` tab 
When i try to get state value from netdaemon app. It works great. 
```cs
        var state = GetState("sensor.testsensor")?.State;
        if (state != null)
        {
            Log(state.ToString());
        }
```

**Comment by helto4real** [comment]
Ahh ok I will look into that after work. I will keep you posted. Thanks again for reporting this! I will check and make sure nothing is broken.

**Comment by helto4real** [comment]
Can confirm this behavour is when running in the add-on but can not reproduce running in dev environment. @ebattulga  can you please confirm it runs as expected in devenvironment?

**Comment by helto4real** [comment]
Fixed in #17. Due to internal name changes in Home Assistant add-on the SetState stopped working. 

See release for details: (178e1bd).

Add-on version fixed: 0.0.34
Nuget Version (make sure to update in dev environment csproj file): 0.1.42-alpha

Keep this issue open for a while incase other have not uppgraded and gets this issue

**Comment by ebattulga** [comment]
Great. Now working successfully.
